ci/slab.toml

```diff
@@ -106,3 +106,9 @@ environment_name = "canada"
 image_name = "Ubuntu Server 22.04 LTS R570 CUDA 12.8"
 flavor_name = "n3-RTX-A6000x1"
 user = "ubuntu"
+
+[backend.hyperstack.4-l40]
+environment_name = "canada"
+image_name = "Ubuntu Server 22.04 LTS R570 CUDA 12.8"
+flavor_name = "n3-L40x4"
+user = "ubuntu"
```

Comment:
Hmm we already have a profile for this
```
[backend.hyperstack.multi-gpu-test]
environment_name = "canada"
image_name = "Ubuntu Server 22.04 LTS R570 CUDA 12.8"
flavor_name = "n3-L40x4"
user = "ubuntu"
```
although it's badly named. I would recommend using the existing profile and in another PR renaming it everywhere (the name dates back to when we only had very few profiles here).